doc/proxy-metrics.md

```diff
@@ -23,8 +23,9 @@ incremented when the response stream ends.
 
 ### `response_latency_ms`
 
-A histogram of the total latency of a response.  This is measured from when the
-request headers are received to when the response stream has completed.
+A histogram of the total latency of a response.  This is measured from when the request
+headers are received to when the first response body frame is received or, if no body
+frames are sent in the stream, to the first EOS frame.
```

Comment:
I think it would be helpful to users to indicate that this is basically "time to first byte" except in the case where the server has sent an empty frame, and except for the fact that we're measuring from the time the request was *received* instead of from the time the request was *forwarded*, so this includes internal overhead of the proxy.

Also, users probably don't understand what "frame" means, especially since only HTTP/2 has frames. I suggest that we explain explicitly what this means for each of (1) HTTP/1 non-chunked transfer encoding, (2) HTTP/1 chunked transfer encoding, and (3) HTTP/2, and avoid using "frame" in the part that isn't specific to HTTP/2.